src/seismometer/builder/resources/classifier_bin.ipynb

```diff
@@ -309,6 +318,7 @@
    },
    "source": [
     "**Tips**: \n",
```

Comment:
I think sphinx/rst requires a blank line for the list to be rendered properly:
![image](https://github.com/epic-open-source/seismometer/assets/36572844/fcf82b31-9a62-49d4-8e13-bd9789f7b105)

A couple of the other tips cells seem affected as well

Reply:
finc